src/main/java/seedu/address/model/tag/UniqueTagList.java

```diff
@@ -47,6 +44,7 @@ public void editTag(Tag oldTag, Tag newTag) {
      * @return True if the tag exists. False if otherwise.
```

Comment:
I believe that there should not be periods `.` at the end of `@return` javadocs.

Reply:
Ok. I made the changes.